app/controllers/public/sessions.js

```diff
@@ -13,6 +14,32 @@ export default class SessionsController extends Controller {
   timezone = null;
   preserveScrollPosition = true;
 
+  @computed('model.session.@each')
+  get groupByDateSessions() {
+    let sessions = groupBy(this.model.session.toArray(), s => moment(s.startsAt).format('dddd, Do MMMM'));
```

Comment:
To fix what @mariobehling mentioned, please use `this.timezone` to convert the date to proper timezone before grouping

Reply:
adding like ```moment(s.startsAt).tz(this.timezone).format('dddd, Do MMMM'))``` give error  
TypeError: _moment.default(...).tz(...) is undefined

Reply:
moment.tz(date, tz)